ui/page/home/view.jsx

```diff
@@ -169,6 +169,8 @@ function HomePage(props: Props) {
           }
         }
         forceShowReposts={id !== 'FOLLOWING'}
+        loading={id === 'FOLLOWING' ? fetchingActiveLivestreams : false}
```

Comment:
This change is "necessary" in order to prevent the shift in the grid when the livestreams get added. So yeah, it's specific to the livestreams. That's currently the most annoying thing on the homepage. Blocklists etc. are also an issue, but those get handled one level deeper for each tile individually. We have to come up with a better solution for those.

Reply:
Ah ok.  The original behavior was to not mount the list at all until livestreams are fetched, but that caused the homepage to be empty for longer than necessary.  This at least places the placeholder tiles :+1:

Reply:
Jep and the site loads MUCH faster than it used to. At least in most regions.